projects/sbb-esta/angular-business/notification/src/notification/notification.component.ts

```diff
@@ -0,0 +1,188 @@
+  /** @docs-private */
+  @HostBinding('class.sbb-notification-error')
+  get typeError(): boolean {
+    return this.type === NotificationType.ERROR;
+  }
+
+  /** @docs-private */
+  @HostBinding('class.sbb-notification-warn')
+  get typeWarn(): boolean {
+    return this.type === NotificationType.WARN;
+  }
+
+  /** @docs-private */
+  @HostBinding('class.sbb-notification-toast-top-left')
+  get positionTopLeft(): boolean {
+    return this.toastPosition === NotificationToastPosition.TOPLEFT;
+  }
+
+  /** @docs-private */
+  @HostBinding('class.sbb-notification-toast-top-right')
+  get positionTopRight(): boolean {
+    return this.toastPosition === NotificationToastPosition.TOPRIGHT;
+  }
+
+  /** @docs-private */
+  @HostBinding('class.sbb-notification-toast-bottom-left')
+  get positionBottomLeft(): boolean {
+    return this.toastPosition === NotificationToastPosition.BOTTOMLEFT;
+  }
+
+  /** @docs-private */
+  @HostBinding('class.sbb-notification-toast-bottom-right')
+  get positionBottomRight(): boolean {
+    return this.toastPosition === NotificationToastPosition.BOTTOMRIGHT;
+  }
+
+  /** Type of notification. */
+  @Input()
+  type: 'success' | 'info' | 'error' | 'warn' = NotificationType.SUCCESS;
+
+  /** Type of notification. */
+  @Input()
+  toastPosition: 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right';
+
+  /** Type of notification. */
+  @Input()
+  readonly = false;
+
+  /** @docs-private */
+  @ViewChild('error', { read: TemplateRef, static: true })
+  errorIcon: TemplateRef<any>;
+
+  /** @docs-private */
+  @ViewChild('check', { read: TemplateRef, static: true })
+  checkIcon: TemplateRef<any>;
+
+  /** @docs-private */
+  @ViewChild('info', { read: TemplateRef, static: true })
+  infoIcon: TemplateRef<any>;
+
+  /** The icon to be used into the notification left side.
+   *  By default uses three icons for SUCCESS, ERROR or INFO notification type,
+   *  but the user can use his own icon using the NotificationIconDirective.
+   */
+  @Input()
+  @ContentChild(IconDirective, { read: TemplateRef, static: false })
+  set icon(notificationIcon: TemplateRef<any>) {
+    this._icon = notificationIcon;
+  }
+
+  get icon() {
+    if (this._icon) {
+      return this._icon;
+    }
+    switch (this.type) {
+      case NotificationType.SUCCESS:
+        return this.checkIcon;
+      case NotificationType.ERROR:
+        return this.errorIcon;
+      case NotificationType.INFO:
+        return this.infoIcon;
+      case NotificationType.WARN:
+        return this.errorIcon;
+      default:
+        return null;
+    }
+  }
+
+  private _icon: TemplateRef<any>;
+
+  /** Title displayed into the notification content */
+  @Input() title: string;
+
+  /** Message displayed into the notification content */
+  @Input() message: string;
+
+  /** List of in page links displayed on the bottom of the notification */
+  @Input() jumpMarks?: JumpMark[];
+
+  @Output()
+  activeChange: EventEmitter<boolean> = new EventEmitter();
```

Comment:
I'm not sure about the naming of this property
It is more a close/dismiss output
Would maybe `dismissed` fit better?
What do you think?

Reply:
Yes, that might be more understandable!